datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/wmi.yaml

```diff
@@ -0,0 +1,20 @@
+- name: host
```

Comment:
Since WMI is mostly deprecated, I really don't think we should have a template

Reply:
I was planning to reuse it on `win32_event_log`

Reply:
I know, we shouldn't though